[2025-01-02T14:02:51.629+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-02T14:02:51.654+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.execute_procedure_task manual__2025-01-02T14:02:39.637448+00:00 [queued]>
[2025-01-02T14:02:51.682+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gold_finance_data_pipeline.execute_procedure_task manual__2025-01-02T14:02:39.637448+00:00 [queued]>
[2025-01-02T14:02:51.683+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2025-01-02T14:02:51.752+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): execute_procedure_task> on 2025-01-02 14:02:39.637448+00:00
[2025-01-02T14:02:51.775+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2553) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-02T14:02:51.778+0000] {standard_task_runner.py:72} INFO - Started process 2555 to run task
[2025-01-02T14:02:51.771+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gold_finance_data_pipeline', 'execute_procedure_task', 'manual__2025-01-02T14:02:39.637448+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/Gold_finance_etl_dags.py', '--cfg-path', '/tmp/tmpet6bjb25']
[2025-01-02T14:02:51.785+0000] {standard_task_runner.py:105} INFO - Job 316: Subtask execute_procedure_task
[2025-01-02T14:02:51.968+0000] {task_command.py:467} INFO - Running <TaskInstance: gold_finance_data_pipeline.execute_procedure_task manual__2025-01-02T14:02:39.637448+00:00 [running]> on host 77bcd09e75b5
[2025-01-02T14:02:52.087+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gold_finance_data_pipeline' AIRFLOW_CTX_TASK_ID='execute_procedure_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T14:02:39.637448+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-02T14:02:39.637448+00:00'
[2025-01-02T14:02:52.089+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-02T14:02:52.124+0000] {base.py:84} INFO - Retrieving connection 'my_postgres_connection'
[2025-01-02T14:02:52.295+0000] {logging_mixin.py:190} INFO - Procedure executed successfully.
[2025-01-02T14:02:52.296+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-02T14:02:52.318+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-02T14:02:52.319+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=gold_finance_data_pipeline, task_id=execute_procedure_task, run_id=manual__2025-01-02T14:02:39.637448+00:00, execution_date=20250102T140239, start_date=20250102T140251, end_date=20250102T140252
[2025-01-02T14:02:52.371+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-02T14:02:52.421+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-02T14:02:52.426+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
